src/core/providers/qgsabstractdatabaseproviderconnection.h

```diff
@@ -195,6 +213,21 @@ class CORE_EXPORT QgsAbstractDatabaseProviderConnection : public QgsAbstractProv
 
     };
 
+    /**
+     * \brief The SqlVectorLayerOptions stores all information required to create a SQL (query) layer.
+     * \see createSqlVectorLayer()
+     *
+     * \since QGIS 3.22
+     */
+    struct CORE_EXPORT SqlVectorLayerOptions
+    {
+      QString sql; //! The SQL expression that defines the SQL (query) layer
```

Comment:
Either
```
      QString sql; //!< The SQL expression that defines the SQL (query) layer
```

or 
```
      //! The SQL expression that defines the SQL (query) layer
      QString sql; 
```

the current formatting will attach the doxygen to the incorrect members